Makefile

```diff
@@ -0,0 +1,34 @@
+
+build: fetch
+	xcodebuild -project ld/zld.xcodeproj -scheme zld -derivedDataPath build -configuration Release build
```

Comment:
why enforce a derived data path?

Reply:
To keep the build products relative to the folder. Easier for the CI jobs and makes it possible to clean without having to invoke xcodebuild.